pvlib/tests/test_pvsystem.py

```diff
@@ -1347,6 +1347,38 @@ def test_PVSystem_pvwatts_ac_kwargs(mocker):
     assert out < pdc
 
 
+def make_irradiance_loss_pvsyst_test_series():
+    final_index = pd.date_range(start='1/1/1990 12:00', periods=365, freq='D')
+    effective_irradiance = pd.Series(1000, index=final_index)
+    shading = pd.Series(.1, index=final_index)
+    snow = pd.Series(.05, index=pd.date_range(start='1/1/1990 12:00',
+                                              periods=365*2, freq='D'))
+    soiling = pd.Series(.02, index=pd.date_range(start='1/1/1990',
+                                                 periods=12, freq='MS'))
+    expected = pd.Series(162.1, index=final_index)
+
+    return {'effective_irradiance': effective_irradiance,
+            'shading': shading, 'snow': snow,
+            'soiling': soiling, 'expected': expected}
+
+
+def test_irradiance_loss_pvsyst():
+    params = make_irradiance_loss_pvsyst_test_series()
+    out = pvsystem.irradiance_loss_pvsyst(params['effective_irradiance'],
+                                          params['shading'], params['snow'],
+                                          params['soiling'])
+    assert_series_equal(params['expected'], out)
```

Comment:
```suggestion
    out = pvsystem.irradiance_loss_pvsyst(
        irradiance_loss_pvsyst['effective_irradiance'], irradiance_loss_pvsyst['shading'],
        irradiance_loss_pvsyst['snow'], irradiance_loss_pvsyst['soiling'])
    assert_series_equal(irradiance_loss_pvsyst['expected'], out)
```

Reply:
Hope I got that right. If so, please change `test_PVSystem_irradiance_loss_pvsyst` to use the fixture.